modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/type/PodcastsSortType.kt

```diff
@@ -89,7 +92,9 @@ enum class PodcastsSortType(
 }
 
 class PodcastsSortTypeMoshiAdapter : JsonAdapter<PodcastsSortType>() {
-    override fun toJson(writer: JsonWriter, value: PodcastsSortType?) {}
+    override fun toJson(writer: JsonWriter, value: PodcastsSortType?) {
+        writer.value(value?.serverId)
+    }
```

Comment:
We never use the PodcastSortType when sending the value to the server but it seems dangerous not to implement it. I have added it just in case we use it in the future.